.github/mergeable.yml

```diff
@@ -0,0 +1,13 @@
+# config file for mergeable: https://github.com/mergeability/mergeable
+version: 2
+mergeable:
+  - when: pull_request.*
+    validate:
+      - do: title
+        must_exclude:
+          regex: ^\[WIP\]
```

Comment:
This is going to miss titles like e.g. `WIP: Something`. Maybe you could instead match
```suggestion
          regex: ^\[?WIP\b
```
It matches the first three but not the last three
```
WIP: stuffing stuffy stuff # Matched
WIP: test                  # Matched
[WIP] Foo bar baz          # Matched

# None of the following match
WIMPs or Dark Matter?
WIPING STUFF OUT
wip: asdf movies
```